app/models/concerns/scoped_search_extensions.rb

```
@@ -4,7 +4,7 @@ module ScopedSearchExtensions
   module ClassMethods
     def value_to_sql(operator, value)
       return value                 if operator !~ /LIKE/i
-      return value.tr_s('%*', '%') if (value ~ /%|\*/)
+      return value.tr_s('%*', '%') if value =~ /%|\*/
```

This was actually silently failing on any "like" search - value is a string and has no method ~